Neos.Flow/Classes/Annotations/Route.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Neos\Flow\Annotations;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * Used to configure Routes from a method
+ *
+ * @Annotation
+ * @Target("METHOD")
+ */
+final class Route
+{
+    /**
+     * Name
+     *
+     * @var string|null
+     */
+    public $name;
+
+    /**
+     * HTTP Methods
+     *
+     * @var array|null
+     */
+    public $httpMethods;
+
+    /**
+     * URI Pattern
```

Comment:
```suggestion
     * URI Pattern. (Can be given as anonymous argument.)
```

Reply:
What does a anonymous argument look like, when annotating a method?

Reply:
```
     * @Flow\Route("path/to/action")
```
instead of
```
     * @Flow\Route(uriPattern="path/to/action")
```
would be possible (while the latter still being allowed).
Also, anonymous arguments can be combined with named arguments:
```
     * @Flow\Route("path/to/action", format="json")
```
It is the equivalent to PHP (8) positional arguments combined with named arguments:
```
function foo($first, $second, $third) { ... }

foo('first-value', second: 'second-value', third: 'third-value');
```
The positional arguments need to come first (though in PHP8 there can be more than one, while in Annotations only one argument can be "anonymous").